elementary/files
# Lose throttle on directory view path-change signals

Fixes #310 

Slot throttles a rapid sequence of path changes from the view but this causes a set of signals to open new tabs or windows to be truncated.  This throttle is now removed since the original reason seems no longer valid.

To test:
Select multiple folders and click to open in multiple tabs (master only opens one tab).
Repeat using context menu.
Repeat using open in new window.

Change path in single tab as rapidly as possible - no crash should occur.
